src/test/java/BaseTest.java

```diff
@@ -8,45 +8,45 @@
 import org.openqa.selenium.remote.RemoteWebDriver;
 import org.openqa.selenium.safari.SafariDriver;
 import org.openqa.selenium.support.ui.WebDriverWait;
-import org.testng.annotations.AfterMethod;
-import org.testng.annotations.BeforeMethod;
-import org.testng.annotations.DataProvider;
+import org.testng.annotations.*;
 
 import java.net.MalformedURLException;
 import java.net.URI;
-import java.net.URL;
 import java.time.Duration;
 
+
 public class BaseTest {
-    private static final ThreadLocal<WebDriver> THREAD_LOCAL = new ThreadLocal<>();
     public static WebDriver driver = null;
+
     public static String url = "https://qa.koel.app/";
     public static WebDriverWait wait = null;
-    public static WebDriver getThreadLocal() {
-        return THREAD_LOCAL.get();
-    }
 
 
+    @BeforeSuite
+    static void setupDriver() {
+        WebDriverManager.chromedriver().setup();
```

Comment:
Please add a line of below code to use the functions of the pick browser method.
`pickBrowser("grid-chrome");`